community-spotlight-streamlit-xata-connector.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Building Streamlit apps faster with the Xata connector'
+description: 'Putting the spotlight on Sergio Demis Lopez Martinez, the developer responsible for a new open source Streamlit : Xata connector.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-streamlit-xata-connector.jpg
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 04-03-2024
+tags: ['community']
+published: true
+slug: community-spotlight-streamlit-xata-connector
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-streamlit-xata-connector.jpg
+---
+
+In the rapidly evolving space of data, analytics and app development [Streamlit](https://streamlit.io/) has risen as a new favorite for fast development of data focused apps. [Sergio Demis Lopez Martinez](https://github.com/SergioSKA27), a 23-year-old student who is currently studying applied mathematics and computer science at the National Autonomous University of Mexico, is one of Streamlit’s biggest fans. His journey through academia and some freelance work has required him to build reliable applications quickly and frequently. This has lead him to choose Streamlit and Xata as his favorite stack for a fast, iterative, development cycle. The use of these two platforms together gave way to a new open source project, [st-xatadb-connection](https://github.com/SergioSKA27/st-xatadb-connection), to enhance the synergy between Xata and Streamlit.
+
+![Streamlit : Xata connector, st-xatadb-connection](/images/st-xatadb-connection.png)
+
+## Xata + Streamlit, the perfect pairing
+
+Both Streamlit and Xata aim to provide a development experience meant to maximize output and reduce delivery time. They work quite well together and enable developers to build great applications quickly without having to worry about the intricacies of the frontend or backend.
+
+Most of the applications Sergio creates in Streamlit are applications focused on data summarization and realtime graphs. Streamlit has popular integrations with Plotly and Matplotlib that make it really easy to create data focused visuals for your application. With PostgreSQL and Elasticsearch underneath the hood, Xata provides aggregations across large volumes of relational data and the tools to slice and dice it — making it a great option for this use case.
+
+## Key features of the st_xata_connection package
+
+The st_xata_connection package has been able to significantly reduce development time for Sergio, sometimes resulting in nearly a 90% reduction in lines of code. For example, this connector automatically saves and caches data with the client so you do not need to make calls to the database as frequently. This can help a lot when dealing with streaming data in real time, you can optimize for larger responses and longer sessions vs. needing to make more frequent calls.
```

Comment:
```suggestion
The st_xatadb_connection package has been able to significantly reduce development time for Sergio, sometimes resulting in nearly a 90% reduction in lines of code. For example, this connector automatically saves and caches data with the client so you do not need to make calls to the database as frequently. This can help a lot when dealing with streaming data in real time, you can optimize for larger responses and longer sessions vs. needing to make more frequent calls.
```